tests/lwip/tcp.c

```diff
@@ -174,7 +174,7 @@ static int tcp_send(char *data, unsigned int num, unsigned int delay)
         else {
             printf("Success: send %u byte over TCP to server\n", (unsigned)data_len);
         }
-        xtimer_usleep(delay);
+        ztimer_sleep(ZTIMER_USEC, delay);
```

Comment:
something like 
d5dee30368b2d21d2a9c5f1bd632e375229ce2b1
is missing here  but this might be an oportunity for another PR

```suggestion
        if( i+1 < num) ztimer_sleep(ZTIMER_USEC, delay);
```

Reply:
see https://github.com/RIOT-OS/RIOT/pull/17115#discussion_r817430688